### ISSUE TITLE
[Rebuild] polkit-dumb-agent-git

### ISSUE THREAD

**Issue by fwookerton** [chaotic-aur/packages#2879]
### Packages:

https://aur.archlinux.org/packages/polkit-dumb-agent-git

### Latest chaotic version

???

### Reason for rebuild

Now I am aware that #2858 mentions this _very_ problem, that the QT/KDE packages are being updated. But, I feel like an interfere to this one won't hurt since _I_ feel this is a crucial package. It's a simple polkit authenticator, which means I depend on it for daily tasks like mounting, unmounting, and doing systemd stuff. Doing an interfere is just as simple as changing the dependency `kdesu` for `kdesu5` and it won't take long enough since it's a very small package anyway. And then when it gets fixed the interfere can be removed.
The problem was already informed to the AUR maintainer and the package is flagged as "out-of-date".

### Logs

https://builds.garudalinux.org/repos/chaotic-aur/logs/polkit-dumb-agent-git.log

### More information

tl;dr an interfere to this package won't hurt.

**Comment by xiota** [comment]
I don't plan to make any kf5-related interferes until late November.  If this is urgent, you can install with `pacman -Sdd`.

The maintainer has had no AUR activity since early 2022.  So I'd suggest opening an orphan request on 2023-11-05 (two weeks after the flag date).  In the request, describe the problems with the package, that the maintainer has been MIA for how long, and why you feel the issue is urgent.

**Comment by xiota** [comment]
Instead of `-Sdd`, better to use `pacman -S --assume-installed [package name]` to create virtual packages to fill the dependencies.

**Comment by xiota** [comment]
@fwookerton Did you resolve this on your local machine or do you still need help with a workaround?

**Comment by fwookerton** [comment]
@xiota Your command works fine and it works as expected, however it looks like this package is not gonna get updated since the maintainer is unreachable and it seems like its gonna stay that way. Better to do an interfere now.

**Comment by Technetium1** [comment]
Related AUR requests:
- [PRQ#50522](https://lists.archlinux.org/hyperkitty/list/aur-requests@lists.archlinux.org/thread/ZNSSXG6ABQO45RAOVUD47OAN7OZWNRS4/#ZNSSXG6ABQO45RAOVUD47OAN7OZWNRS4)
- [PRQ#50480](https://lists.archlinux.org/hyperkitty/list/aur-requests@lists.archlinux.org/thread/O4XTATAYNN2Q7DNPHOUYOQAEKAQE5KHM/#RQEZJVKECIRKKANDNRXB226GCYQDZ4EH)

**Comment by xiota** [comment]
There is `chaotic-kf5-dummy` to work around this issue for now.  Will reevaluate whether to write interferes to drop packages when it is time to drop the dummy package.

**Comment by fwookerton** [comment]
Thanks, it looks like this package doesn't get the care it needs.. However, it's one of the simpler polkit auths out there, i dunno if there are other minimalist authenticators like this one, if there is I might consider changing to that.
I don't want to install KDE's or GNOME's polkits bc they install a billion packages i don't need.

**Comment by xiota** [comment]
Dropping because incompatible with KF6.  Upstream project appears abandoned.

**Comment by fluffeon** [comment]
Why was KF5 dropped?

**Comment by dr460nf1r3** [comment]
There is no more kf5?

**Comment by xiota** [comment]
> Why was KF5 dropped?

The Arch way... bleeding edge for stuff that's broken (KF6)... out of date for stuff that isn't (electron, clang, other libraries).
